resolwe_bio/variants/models.py

```diff
@@ -0,0 +1,151 @@
+""".. Ignore pydocstyle D400.
+
+======
+Models
+======
+
+"""
+
+from django.conf import settings
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from resolwe.flow.models import Data
+from resolwe.flow.models import Entity as Sample
+
+# TODO: sync with the kb, at least the first entry.
+SPECIES_MAX_LENGTH = 50
+GENOME_ASSEMBLY_MAX_LENGTH = 20
+CHROMOSOME_MAX_LENGTH = 20
+REFERENCE_MAX_LENGTH = 100
+ALTERNATIVE_MAX_LENGTH = 100
+
+# Metadata
+VARIANT_DATA_SOURCE_MAX_LENGTH = 100
+VARIANT_ANNOTATION_SOURCE_MAX_LENGTH = 100
+ANNOTATION_MAX_LENGTH = 200
+GENOTYPE_MAX_LENGTH = 100
```

Comment:
If this refers to genotypes in format "0/1" or "1/1", it could also be shortened.

Reply:
Good point. Currently, the mutations_table output for GT field is more verbose, for example G/T, T/T, TAA/T. This then gets converted to 0, 1 or 2 format in [VariantsTable](https://github.com/genialis/resolwe-bio-py/blob/master/src/resdk/tables/variant.py#L350). We should do this encoding when writing to DB, and keep the length of this field at max 3 (so that either 0/1 or 1 format is supported).

Reply:
I changed the length to 3. If is also easy to increase the size in the future if required.